django/db/models/indexes.py

```diff
@@ -12,9 +14,13 @@ class Index:
     # cross-database compatibility with Oracle)
     max_name_length = 30
 
-    def __init__(self, *, fields=(), name=None, db_tablespace=None, opclasses=()):
+    def __init__(self, *, fields=(), name=None, db_tablespace=None, opclasses=(), conditions=Q()):
```

Comment:
I think you should use `None` as a default value here to avoid having potentially mutable default.

Reply:
I had a look at `Aggregate` and the default value there. I'm fine with `None` as the default.